src/lucky_web/version.cr

```diff
@@ -1,3 +1,3 @@
 module LuckyWeb
-  VERSION = "0.4.2"
+  VERSION = "0.4.3"
```

Comment:
Since this is a breaking change, normally the major version number would change, but since this is pre-release I didn't know what to change this to, if anything. Happy to do whatever.

Reply:
I think you could leave this as-is. I'll bump it to 0.5.0 once a couple more things are merged